main.py

```diff
@@ -470,9 +501,9 @@ def main() -> None:
     parser.add_argument(
         "-c",
         "--concurrency",
-        default=CPU_COUNT,
+        default=CPU_COUNT // 2,
         type=int,
-        help="number of concurrent workers (defaults to number of CPU cores)",
+        help="number of concurrent workers (defaults to half the number of CPU cores), use -1 to use all CPU cores",
```

Comment:
What's wrong with defaulting to all cores? Not a fan of magic like "-1 means all cores"

Reply:
I changed the behavior back to default all CPU cores.
The reason I did this was because it now eats CPU for breakfast and no longer just hovers around 30% usage.
Normally I would just keep doing other things while test262 ran but now I can no longer do that. But people can still just pass `-c XXX` if they want so it's fine.